### PR TITLE
Review

### PR DESCRIPTION
**Pull request by dennisli92**
(bit of a hack to do a PR on the entire commit history :) http://astrofrog.github.io/blog/2013/04/10/how-to-conduct-a-full-code-review-on-github/)

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (dennisli92) [inline public/manifest.json]: Low priority, but might be worth either updating this (Or just deleting it)

**Comment by dennisli92** [comment]
I don't have many strong opinions on this, esp since it's mostly markup. but it'd be good to get an idea of how the files are organized/designs you had in mind. e.g. it seems like the "pages" are all in `src/` and then use components in `src/components/`. I'm just unclear on why sometimes the pages are just very thin wrappers around the components, and other times, involve more markup.

**Comment by budimirbudimir** [comment]
@dennisli92 I actually have no clue what we planned on the website, but I guess we can't go wrong with Redux? I'll see to reorganize these components made on-the-fly to be better structured. And edit manifest. :)

**Comment by dennisli92** [comment]
I think redux is unnecessary since it's all static content (I believe?)
just curious how you were organizing it.

**Pull request merged**